File: giliboim-frontend/src/components/types/ChatComment.ts
// src/types/Chat.ts
export interface ChatRoom {
    id: number;
    name: string;
}

export interface ChatComment {
    chatCommentNo: number;
    chatRoomNo: number;
    memberNo: number;
    chatText: string;
    createDate: string; // ISO 형식의 날짜 문자열
}

File: giliboim-backend/src/main/java/com/kh/giliboim/personal/model/vo/ChatComment.java
package com.kh.giliboim.personal.model.vo;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChatComment {

	private int chatCommentNo;
	private int chatRoomNo;
	private int memberNo;
	private String chatText;
	private String nickname;
	private String createDate; 
}
